assets/css/less/twenty-twenty-one.less

```diff
@@ -0,0 +1,17 @@
+.wpbdp-wp-theme-twentytwentyone {
+    .wpbdp-main-box {
+        .submit-btn input[type="submit"] {
+            padding: 12px;
```

Comment:
14px 12px looks a bit better for me. This makes the fields and buttons line up.

Reply:
In your screenshot, the find listings button is a little taller than the input. With padding: 14px 12px, it lines up vertically too.

Reply:
@theRealRizeo this is too short now and isn't lining up vertically... Please make sure they line up perfectly since we're overriding the theme styling here.